compiler/src/main/java/org/qbicc/graph/ValueHandleVisitorLong.java

```diff
@@ -0,0 +1,181 @@
+package org.qbicc.graph;
+
+/**
+ *
+ */
+public interface ValueHandleVisitorLong<T> {
```

Comment:
I didn't see where this new visitor was used.  What am I missing?

Reply:
It's used by `org.qbicc.interpreter.impl.Frame#getOffset` to get a numeric result for a handle (in this case, what the offset is for the member compared to the base of the `Memory` object) without having to construct a box for the value.

Reply:
Got it; thanks!  Missed that in the 2132 lines & GitHub had helpfully elided the details of the big diffs when I was doing a search.